src/js/app/components/form/input/input.pug

```diff
@@ -12,23 +12,23 @@
         </div>
 
         //- text input
-        <input v-if="type === 'text'" type="text" class="input input--text" :id="`input-${id}`" v-bind="props" v-model="data" :disabled="disabled" :readonly="disabled" />
+        <input v-if="type === 'text'" :data-cy-field="id" type="text" class="input input--text" :id="`input-${id}`" v-bind="props" v-model="data" :disabled="disabled" :readonly="disabled" />
```

Comment:
Petite remarque ou je n'ai pas vraiment d'avis tranché dessus, le fait de seletionner les inputs via `data-cy-field` a l'inconvénient de devoir rajouter un peu de boilerplate code à tous les inputs. Je me demande si on ne pouvait pas se baser uniquement sur des ids vu qu'ils sont toujours définis ici il me semble pour la gestion des erreurs

Reply:
Nvm, je viens de voir que c'est ce que conseille cypress, je sais pas si je suis totalement convaincu du coté volatil des ids dans notre cas mais why not 👍  https://docs.cypress.io/guides/references/best-practices.html#How-It-Works